vouch/controllers/metrics_controller.py

```diff
@@ -52,9 +105,11 @@ def get(self):
         """
         try:
             self.get_cert_expiration_time()
+            host_signing_token_expiry = self.get_host_signing_token_expiry()
             ca_cert_expiry_time = time.mktime(self.cert_expiration_time.timetuple())
             current_time = time.time()
             g_ca_cert_expiry_time.set(int(ca_cert_expiry_time))
+            g_host_signing_token_expiry.set(host_signing_token_expiry)
```

Comment:
get_host_signing_token_expiry func returns `None` in some cases. Does this set func allow setting a `None` value? if yes, (as mentioned in earlier comment as well) what do we see in prom?
maybe good to typecast this again to int before setting here, similar to `g_ca_cert_expiry_time.set(int(ca_cert_expiry_time))`?